nptdms/base_segment.py

```diff
@@ -56,14 +56,14 @@ def _read_data_chunk(self, file, data_objects, chunk_index):
         """
         raise NotImplementedError("Data chunk reading must be implemented in base classes")
 
-    def read_channel_data_chunks(self, file, data_objects, channel_path, chunk_offset, stop_chunk):
+    def read_channel_data_chunks(self, file, data_objects, channel_path, chunk_offset, stop_chunk, chunk_size):
```

Comment:
Sorry I missed this earlier, but it looks like these extra `chunk_size` parameters aren't needed? Only the new parameter in `_read_channel_data_chunks` is used.